osu.Game.Rulesets.Osu/Edit/OsuSelectionHandler.cs

```diff
@@ -97,6 +103,20 @@ public override bool HandleReverse()
                 }
             }
 
+            foreach (var h in hitObjects)
+            {
+                if (h is OsuHitObject obj) obj.NewCombo = false;
+            }
+
+            foreach (double place in newComboPlaces)
+            {
+                hitObjects
+                    .OfType<OsuHitObject>()
+                    .Where(obj => obj.StartTime == place)
```

Comment:
I'm feeling uneasy about using float equality here because the code above calculates `StartTime` rather than for example transferring it across directly, meaning the calculation could result in FP errors that evade this equality check.

Reply:
Fixed the logic, now it saves the order in which new combos are set [True, False, False] (in the example of the video), reverts the pattern and reapply the combo flags to each one of the HitObjects